addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/RemoteFileIncludeScanRule.java

```diff
@@ -87,8 +87,8 @@ public class RemoteFileIncludeScanRule extends AbstractAppParamPlugin
         Pattern.compile("<title>Google</title>"),
         Pattern.compile("<title>Google</title>"),
         Pattern.compile("<title>Google</title>"),
-        Pattern.compile("<title.*?Google.*?/title>"),
-        Pattern.compile("<title.*?Google.*?/title>"),
+        Pattern.compile("<title.*ZAP.*Google.*/title>"),
```

Comment:
Why now greedy?

Reply:
And isn't this still going to lead to FPs if the title already has ZAP+Google like before when it had only Google? I'd have expected changes around that logic.

Reply:
I figured since ZAP was being sent as a search term that it was sufficiently unlikely to happen. I mean I guess one of our future GSoC students may get hit it or something but "in the wild" I felt like it was likely okay.

As for the greedy change that was just me not having read up on things. I was thinking it was two quantifies.

Reply:
For the record the ZAP+Google was meant as combination of the two, not a verbatim string.

Reply:
I will leave up to you whether to keep it or not.

Reply:
I don't think it hurts to leave it.